pkg/controller/directvolumemigration/rsync.go

```diff
@@ -1611,12 +1611,35 @@ func (req rsyncClientPodRequirements) getRsyncClientPodTemplate() corev1.Pod {
 	trueBool := true
 	isPrivileged := req.privileged
 	volumes := []corev1.Volume{}
-	volumeMounts := []corev1.VolumeMount{}
+	rsyncVolumeMounts := []corev1.VolumeMount{}
 	containers := []corev1.Container{}
-	volumeMounts = append(volumeMounts, corev1.VolumeMount{
+	rsyncVolumeMounts = append(rsyncVolumeMounts, corev1.VolumeMount{
 		Name:      req.pvInfo.name,
 		MountPath: fmt.Sprintf("/mnt/%s/%s", req.namespace, req.pvInfo.name),
 	})
+
+	// shared volumeMount for inter-process communication between rsync and stunnel
+	rsyncVolumeMounts = append(rsyncVolumeMounts, corev1.VolumeMount{
+		Name:      "rsync-stunnel-ipc",
+		MountPath: "/usr/share/rsync-data",
+	})
+
+	stunnelVolumeMounts := []corev1.VolumeMount{
+		{
+			Name:      "stunnel-conf",
+			MountPath: "/etc/stunnel/stunnel.conf",
+			SubPath:   "stunnel.conf",
+		},
+		{
+			Name:      "stunnel-certs",
+			MountPath: "/etc/stunnel/certs",
+		},
+		{
+			Name:      "rsync-stunnel-ipc",
+			MountPath: "/rsync-data",
```

Comment:
should this be `/usr/share/rsync-data`?

Reply:
w.r.t functionality it does not matter but good to keep that same. That way we don't have to look at the YAML everytime we are debugging this

Reply:
Done updated it to `/usr/share/rsync-stunnel-mgmt` for uniformity.